FormalConjectures/Wikipedia/RamanujanTau.lean

```diff
@@ -32,9 +32,9 @@ There are two conjectures related to the Ramanujan τ-function:
 
 open PowerSeries PowerSeries.WithPiTopology
 
-noncomputable def Δ : PowerSeries ℤ := X * ∏' (n : ℕ+), (1 - X ^ (n : ℕ)) ^ 24
+private noncomputable def Δ : PowerSeries ℤ := X * ∏' (n : ℕ+), (1 - X ^ (n : ℕ)) ^ 24
```

Comment:
I wonder if we shouldn't just lint for non-private definitions...

Reply:
Would be a good idea, I think we rarely re-use definitions from other files and prefer to include them in ForMathlib